app/commands/exercise/representation/search.rb

```diff
@@ -10,7 +10,9 @@ def self.requests_per_page = 20
   end
 
   def call
-    @representations = Exercise::Representation.includes(:exercise, :track)
+    @representations = Exercise::Representation.
+      includes(:exercise, :track).
+      where('num_submissions > 1')
```

Comment:
How does this work with indexing?

Reply:
On filtering its faster by about 80ms. On count, it's slower by about 2ms.

🤷

Reply:
Okay.

Reply:
Do we need to update our indexes?